.github/workflows/docker-deploy.yaml

```
@@ -18,21 +25,14 @@ jobs:
-          docker tag ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:${{ env.DATE_TAG }} ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:${{ env.DATE_TAG }}
-          # Tag as latest
-          docker tag ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:${{ env.DATE_TAG }} ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:latest
+        run: docker tag ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.IMAGE_NAME }}:${{ env.TAG }} ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.IMAGE_NAME }}:latest
 
       - name: Push the Docker image
         run: |
-          docker push ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:${{ env.DATE_TAG }}
-          docker push ${{ secrets.DOCKERHUB_USERNAME }}/electronyat-shop:latest
\ No newline at end of file
+          docker push ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.IMAGE_NAME }}:${{ env.TAG }}
+          docker push ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.IMAGE_NAME }}:latest
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)